project-reports/2023/2023-Q2-Hyperledger-Sawtooth.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Sawtooth
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Sawtooth is going through a transition from Transact being separated and the idea that Sawtooth 2.0 would use https://github.com/Cargill/splinter . The last two monthly calls have looked at merging Transact and some of the libraries Bitwise and Blockchain TP created with the current 1.3.x branch and renaming it "main".  That would allow the 1.x branch to continue as several companies are using it exclusively for their clients(including Taekion & BlockchainTP)as well as not be dependent on a non-Hyperledger project - Splinter. The advantage is having a blockchain that can easily using different consensus mechanisms easily for different workload situations. 
```

Comment:
[nitpick] The advantage ... that can `[adopt to a wide range of]` consensus mechanisms easily for different workloads as the situation demands.